src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -20,15 +20,15 @@ import styles from './BrowserTrackConfig.scss';
 
 import SlideToggle from 'src/shared/components/slide-toggle/SlideToggle';
 
-import trackHeightBtn from 'static/img/browser/icon_tracks_height_grey.svg';
-import trackLockBtn from 'static/img/browser/icon_tracks_lock_open_grey.svg';
-import trackHighlightBtn from 'static/img/browser/icon_tracks_highlight_grey.svg';
-import trackMoveBtn from 'static/img/browser/icon_tracks_move_grey.svg';
 import { RootState } from 'src/store';
 import { CogList } from '../browserState';
 import Checkbox from 'src/shared/components/checkbox/Checkbox';
 import useOutsideClick from 'src/shared/hooks/useOutsideClick';
 
+import { browserTrackConfig } from '../browserConfig';
+import ImageButton from 'src/shared/components/image-button/ImageButton';
```

Comment:
The imports here also need to be reordered. `ImageButton` and `Checkbox` will need to go below `SlideToggle` and the `styles` will need to come to the bottom of the imports list.

Reply:
Done.